### ISSUE TITLE
InlinePanel doesn't take a classname

### ISSUE THREAD

**Issue by emg36** [torchbox/wagtail#1316]
InlinePanels don't seem to accept classnames as the documents suggest. Its seems it has been omitted from being popped with the other kwargs in edit_handlers. I have suggested 2 lines below to fix this but maybe it was just a mistake in the docs.

```
class InlinePanel(object):
    def __init__(self, *args, **kwargs):
        # prior to Wagtail 0.9, InlinePanel required two params, base_model and relation_name.
        # base_model is no longer required; we set up relations based on the model passed to
        # bind_to_model instead
        if len(args) == 1:  # new-style: InlinePanel(relation_name)
            self.relation_name = args[0]
        elif len(args) == 2:  # old-style: InlinePanel(base_model, relation_name)
            self.relation_name = args[1]

            warnings.warn(
                "InlinePanel no longer needs to be passed a model parameter. "
                "InlinePanel({classname}, '{relname}') should be changed to InlinePanel('{relname}')".format(
                    classname=args[0].__name__, relname=self.relation_name
                ), RemovedInWagtail12Warning, stacklevel=2)
        else:
            raise TypeError("InlinePanel() takes exactly 1 argument (%d given)" % len(args))

        self.panels = kwargs.pop('panels', None)
        self.label = kwargs.pop('label', '')
        self.help_text = kwargs.pop('help_text', '')
        self.classname = kwargs.pop('classname', '') <-----------------------

        if kwargs:
            raise TypeError("InlinePanel got an unexpected keyword argument '%s'" % kwargs.keys()[0])

    def bind_to_model(self, model):
        return type(str('_InlinePanel'), (BaseInlinePanel,), {
            'model': model,
            'relation_name': self.relation_name,
            'related': getattr(model, self.relation_name).related,
            'panels': self.panels,
            'heading': self.label,
            'help_text': self.help_text,  # TODO: can we pick this out of the foreign key definition as an alternative? (with a bit of help from the inlineformset object, as we do for label/heading)
            'classname': self.classname, <-----------------------
      })
```

**Comment by robslotboom** [comment]
According to the current (1.6.2) source this issue is still valid.

```
class InlinePanel(object):
    def __init__(self, relation_name, panels=None, label='', help_text='', min_num=None, max_num=None):
        self.relation_name = relation_name
        self.panels = panels
        self.label = label
        self.help_text = help_text
        self.min_num = min_num
        self.max_num = max_num

    def bind_to_model(self, model):
        if django.VERSION >= (1, 9):
            related = getattr(model, self.relation_name).rel
        else:
            related = getattr(model, self.relation_name).related

        return type(str('_InlinePanel'), (BaseInlinePanel,), {
            'model': model,
            'relation_name': self.relation_name,
            'related': related,
            'panels': self.panels,
            'heading': self.label,
            'help_text': self.help_text,
            # TODO: can we pick this out of the foreign key definition as an alternative?
            # (with a bit of help from the inlineformset object, as we do for label/heading)
            'min_num': self.min_num,
            'max_num': self.max_num
        })


```

**Comment by gasman** [comment]
Apologies, it looks like we overlooked this issue originally...

Fix now applied in b0aa210078fa3be5d7ccbb86214c3fe5db8568d8 - thanks!
